packages/tsutils/src/datetime.ts

```diff
@@ -6,7 +6,7 @@
 import moment from 'moment';
 import momentTimezone from 'moment-timezone';
 
-export const ISO_DATE_PATTERN = /^\d{4}-\d{2}-\d{2}$/;
+export const ISO_DATE_PATTERN = /\d{4}-\d{2}-\d{2}/;
```

Comment:
This also works:

```suggestion
export const ISO_DATE_PATTERN = /^\d{4}-\d{2}-\d{2}/;
```

Feel free to commit the suggestion directly if you prefer it